src/features/analytics_panel/constants.ts

```diff
@@ -1 +1,2 @@
 export const MIN_HEIGHT = 119;
+export const DESIRED_HEIGHT = 320
```

Comment:
DESIRED_HEIGHT need to calculate from content, it dynamic one (326.667px for example)